### ISSUE TITLE
Rendering of dashes causes problems

### ISSUE THREAD

**Issue by rurseekatze** [rurseekatze/node-tileserver#49]
Currently the rendering of dashes does not work in every case. 

Example: Süntelbahn http://www.openrailwaymap.org/canvas.php?lang=de&lat=52.197007066922616&lon=9.450516700744629&zoom=14&style=standard

![abandoned-join](https://cloud.githubusercontent.com/assets/10909049/8891118/d2ecf8ee-331b-11e5-93d1-8397d1c9a3d0.png)

On zoom 15 it get's worse, but I suspect this could be the renderer doing something wrong:
![dashes-5-6-z15](https://cloud.githubusercontent.com/assets/10909049/9289268/8b35df84-436a-11e5-88d5-77c4beee8bf2.png)

On zoom 16 it's all fine:
![dashes-5-6-z16](https://cloud.githubusercontent.com/assets/10909049/9289278/e9a59d70-436a-11e5-8dd5-8f915b39624b.png)

**Comment by DerDakon** [comment]
I think what happens here is that the line is acutally drawn twice, with a slight offset.

**Comment by DerDakon** [comment]
The line is drawn twice, but to me it looks like node-tileserver is not to blame here. When you look on the data that is rendered, which is [tile 15/17243/10795](http://a.tiles.openrailwaymap.org/vector/15/17243/10795.js), you see that nearly the same way is twice in the list. One comes from the relation, which is errnously also tagged as route=railway;railway=abandoned, the other one is the way itself.

I don't understand:
 * why the part of the relation appears as simple way here
 * why the part of the relation is split differently than the way itself, which in OSM data is long enough to cross the tile border

Since I fixed the relation, I've also attached the data as it was rendered today: 
[10795.js.txt](https://github.com/rurseekatze/node-tileserver/files/129638/10795.js.txt)
